plugins/woocommerce-admin/client/customize-store/assembler-hub/style.scss

```diff
@@ -445,7 +446,7 @@
 	/* Layout sidebar */
 	.block-editor-block-patterns-list__item {
 		.block-editor-block-preview__container {
-			border-radius: 2px;
+			border-radius: 4px;
```

Comment:
Oh I think this might have been accidentally included, as you have a PR for it here https://github.com/woocommerce/woocommerce/issues/40904